spec/examples/time_util_spec.rb

```diff
@@ -92,6 +92,12 @@ module IceCube
       end
     end
 
+    describe :serialize_time do
+      it "supports ISO8601 time strings" do
+        expect(TimeUtil.serialize_time(Time.utc(2014, 4, 4, 10, 30, 0))).to eq("2014-04-04T18:30:00+08:00")
+      end
+    end
```

Comment:
Suggestion to make the specs clearer:
```suggestion
    describe :serialize_time do
      subject(:serialize_time) { TimeUtil.serialize_time(time) }

      let(:time) { Time.utc(2014, 4, 4, 10, 30, 0) }
      let(:iso_time_str) { "2014-04-04T18:30:00+08:00" }

      it "supports ISO8601 time strings" do
        expect(serialize_time).to eq(iso_time_str)
      end
    end
```